### PR TITLE
Patch/AdminController NumberFormatException + minor fix to admin.jsp

### PR DESCRIPTION
**Pull request by toneo-**
This fixes the crash that happens whenever you go to the admin page, which must have slipped past a previous code review.

The problem was the JSP had been designed to expect List<Member>, but was actually being given List<Object[]>, which it was getting from MembersService. Since MembersRepo already provides the functionality we need and MembersService doesn't do anything else I've deleted MembersService.

Also noticed admin.jsp got your username wrong, so now it displays it properly

### CODE REVIEW

File: XYZDrivers/src/main/java/com/xyzdrivers/services/InsertPaymentService.java [hunk 1]
```diff
@@ -71,29 +71,29 @@ public void InsertPayment(MembershipPayment mp) throws IllegalAccessException, S
...
+//        String query2 = "SELECT BALANCE FROM MEMBERS WHERE ID = ?";
+//
+//        PreparedStatement ps2 = con.prepareStatement(query2);
+//
+//        ps2.setString(1, mp.getMemberID());
+//
+//        results = ps2.executeQuery();
+//
+//        while (results.next()) {
+//            currentBalance = results.getFloat("BALANCE");
```
Comment (bkneis) [inline XYZDrivers/src/main/java/com/xyzdrivers/services/InsertPaymentService.java]: please delete commented code if not being used

**Review by bkneis — CHANGES_REQUESTED** [review]
Great job, good catch on the bugs. Just one comment

File: XYZDrivers/src/main/java/com/xyzdrivers/services/InsertPaymentService.java (hunk 1 above)
Comment (toneo-) [reply XYZDrivers/src/main/java/com/xyzdrivers/services/InsertPaymentService.java]: Deleted, also deleted in #24 .

**Comment by toneo-** [comment]
Changes made, merge conflicts fixed

**Review by bkneis — APPROVED** [review]
LGTM

**Pull request merged**